spec/view_integration/post_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts Workflow', type: :feature do
+  let!(:user) { create(:user, posts_counter: 3) }
+  let!(:post) { create(:post, author: user, comments_counter: 2, likes_counter: 2) }
+
+  describe 'User post index page' do
+    it 'displays user information, profile picture, and post count' do
+      visit user_posts_path(user)
+      expect(page).to have_content(user.name)
+      expect(page).to have_selector("img[src='#{user.photo}']")
+    end
+
+    it 'displays post title, body, and comments' do
+      visit user_posts_path(user)
+      expect(page).to have_content(post.title)
+      expect(page).to have_content(post.text)
+      expect(page).to have_content(post.comments_counter)
```

Comment:
- Good job done so far, but according to the project requirements, you should have a test that asserts that you can see the first comments on a post. I suggest you use the method that returns the 3 most recent comments to do this. Please fix this 🙏 
- According to the project requirements, you should have a test that asserts you can see the number of likes a post has. Please fix this 🙏